src/sentry/tasks/store.py

```diff
@@ -280,7 +280,7 @@ def save_event(cache_key=None, data=None, start_time=None, event_id=None, **kwar
     except HashDiscarded as exc:
         info_logger.info(
```

Comment:
Do we not ever need to know about these events? This isn't really my call to make since I didn't add this. :) @macqueen